frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/commentsPanel/commentsPanel.component.tsx

```diff
@@ -47,9 +64,8 @@ export const CommentsPanel = ({ scrollPanelIntoView }: CommentsPanelProps) => {
 	const isFederation = modelIsFederation(containerOrFederation);
 	const ticketId = TicketsCardHooksSelectors.selectSelectedTicketId();
 	const comments = TicketCommentsHooksSelectors.selectComments(ticketId);
-	const scrollAreaRef = useRef<Scrollbars>();
 
-	const commentsListIsEmpty = comments?.length > 0;
+	const commentsLength = comments?.length || 0;
```

Comment:
I agree, it may be omitted and the component would still work, but the name indicates a number. If comments is undefined,  so would the length. I'd rather keep the `|| 0` tbh

Reply:
but won't comments always be defined since selectComments returns [] if undefined?

Reply:
Ah fair point, I didn't realise that. In that case I'll proceed to change the code accordingly